tests/framework/e2e/etcd_process.go

```diff
@@ -163,12 +162,6 @@ func (ep *EtcdServerProcess) Close() error {
 	return nil
 }
 
-func (ep *EtcdServerProcess) WithStopSignal(sig os.Signal) os.Signal {
```

Comment:
You can't remove this method directly. 

```
 stderr: # go.etcd.io/etcd/tests/v3/framework/e2e
Error: stderr: framework/e2e/cluster_proxy.go:105:19: p.etcdProc.WithStopSignal undefined (type EtcdProcess has no field or method WithStopSignal)
Error: stderr: framework/e2e/cluster_proxy.go:158:16: pp.proc.StopSignal undefined (type *expect.ExpectProcess has no field or method StopSignal)
Error: stderr: framework/e2e/cluster_proxy.go:159:9: pp.proc.StopSignal undefined (type *expect.ExpectProcess has no field or method StopSignal)

```

Reply:
Missed this due to `proxy` having a different build target :(